packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.Props.ts

```diff
@@ -105,5 +104,9 @@ export interface ISpinButtonProps {
 }
 
 export interface ISpinButton {
-  value?: number;
+  /**
+   * The value of the SpinButton. Use this if you intend to pass in a new value as a result of onChange events.
```

Comment:
I don't know if this description matches the intention of this, can you set the value from here? I thought this was for readonly when using the component as uncontrolled component.